queries/highlights.scm

```diff
@@ -0,0 +1,43 @@
+(doctype) @constant
+
+((attribute_name) @keyword
+ (#match? @keyword "^phx-"))
```

Comment:
I thought that this might be interesting based on the behavior of https://github.com/Ritvyk/vscode-heex-html

Maybe it also makes sense to add javascript injections to attribute-values for attributes that are usually alpine.js? On the other hand, alpine.js isn't a built-in part of heex or live-view so maybe it'd just be confusing

Reply:
> Maybe it also makes sense to add javascript injections to attribute-values for attributes that are usually alpine.js? On the other hand, alpine.js isn't a built-in part of heex or live-view so maybe it'd just be confusing

I know with nvim-treesitter you can supply your own additional highlight queries, does Helix support that as well? I don't think this grammar should include anything that is not part of HEEx itself.

Reply:
It does! I think I might end up doing this injection for the Helix queries but it probably makes less sense for the in-repo ones (I think it'd be weird to see those get highlighted on GitHub)

> I don't think this grammar should include anything that is not part of HEEx itself.

I agree, that makes sense :+1: